modules/migration-upgrading-migration-tool.adoc

```diff
@@ -26,6 +27,40 @@ The following procedure enables you to change the *Manual* approval option to *A
 . Optional: Edit the *Channel*.
 +
 Updating the subscription deploys the updated {mtc-operator} and updates the {mtc-short} components.
+endif::[]
+ifeval::["{mtc-version}" == "1.3"]
+You can upgrade the Cluster Application Migration tool to {mtc-short} on an {product-title} 4 cluster by uninstalling the *CAM Operator* and installing the {mtc-operator}.
+
+.Procedure
+
+. In the {product-title} console, navigate to *Operators* > *Installed Operators*.
+. Select the *CAM Operator*.
+. On the right-hand side of the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu.
++
+An *Uninstall Operator?* dialog box is displayed, reminding you that:
++
+[.small]
+--
+*Removing the Operator will not remove any of its custom resource definitions or
```

Comment:
If this is exactly the UI text, it's fine. If it's not, remove the future tense and "need to."